Main.java

```diff
@@ -1,121 +1,255 @@
+                    float d = scan.nextFloat();
+                    Trapezoid t = new Trapezoid(a, b, c, d);
+                    System.out.printf("The perimeter of the Trapezoid with First Side: %.2f, Second Side: %.2f, Third Side: %.2f, and Fourth Side: %.2f is %.2f", a, b, c, d, t.calculatePerimeter());
+                    break;
+
+                case 'X':
+                    System.out.println("Calculation for rhombus");
+                    float ra = scan.nextFloat();
+                    System.out.println("Input second side");
+                    float rb = scan.nextFloat();
+                    System.out.println("Input third side");
+                    float rc = scan.nextFloat();
+                    System.out.println("Input fourth side");
+                    float rd = scan.nextFloat();
+                    System.out.println("Input first diagonal");
+                    float rd1 = scan.nextFloat();
+                    System.out.println("Input second diagonal");
+                    float rd2 = scan.nextFloat();
+
+                    Rhombus rh= new Rhombus(ra, rb, rc, rd,rd1,rd2);
+
+                    System.out.printf("The perimeter of the rhombus with First Side: %.2f, Second Side: %.2f, Third Side: %.2f, Fourth Side: %.2f,first diagonal %.2f and second diagonal %.2f is %.2f", ra, rb, rc, rd, rd1, rd2, rh.calculatePerimeter());
+                    System.out.println("");
+                    System.out.printf("The area of the rhombus with First Side: %.2f, Second Side: %.2f, Third Side: %.2f, Fourth Side: %.2f,first diagonal %.2f and second diagonal %.2f is %.2f", ra, rb, rc, rd, rd1, rd2, rh.area());
+                    break;
+
+                case 'S':
+                    System.out.println("Calculation for a Square");
+                    System.out.println("Input the Length");
+                    float xy = scan.nextFloat();
+                    Square s = new Square(xy);
+                    System.out.printf("The perimeter of the square with  Side: %.2f is %.2f", xy, s.calcPerimeter());
+                    System.out.printf("\nThe area of the square with Side: %.2f is %.2f", xy, s.area());
+                    break;
+
+                case 'R':
+                    System.out.println("Calculation for a Rectangle");
+                    System.out.println("Input length");
+                    float recL = scan.nextFloat();
+                    System.out.println("Input breadth");
+                    float recB = scan.nextFloat();
+                    Rectangle rec = new Rectangle(recL,recB);
+                    System.out.printf("The perimeter of the rectangle with  length: %.2f and breadth %.2f is %.2f", recL,recB, rec.perimeter());
+                    System.out.println("");
+                    System.out.printf("The area of the rectangle with length: %.2f and breadth %.2f is %.2f", recL,recB, rec.area());
+                    break;
+
+                default:
```

Comment:
you should have giving the user an information to show that s/he chose the wrong option